_targets.R

```diff
@@ -7,9 +7,34 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
```

Comment:
One small tweak for additional best practices. For arguments that we'll be using for many targets, it's best practice to define these at the top of the `_targets.R` file, so that if we ever want to change them, we can do so in one spot only, and the change percolates through to all of the targets that use those arguments. So you can define  them here, like so:
```r
parameter_code = '00010'
start_date = "2014-05-01"
end_date = "2015-05-01"
```

and then in your targets you could reference them like:

```r
tar_target(
    site_data_01427207_csv,
    download_nwis_site_data("01427207", out_file="1_fetch/tmp/nwis_01427207_data.csv", parameterCd = parameter_code, startDate=start_date, endDate=end_date),
    format = "file"
  ),
```